rust/threshold-ecdsa/src/ecdsa_example_rust/src/lib.rs

```diff
@@ -104,25 +63,19 @@ async fn verify(
     message: String,
     public_key_hex: String,
 ) -> Result<SignatureVerificationReply, String> {
-    let signature_bytes = hex::decode(&signature_hex).expect("failed to hex-decode signature");
-    let pubkey_bytes = hex::decode(&public_key_hex).expect("failed to hex-decode public key");
+    let signature_bytes = hex::decode(signature_hex).expect("failed to hex-decode signature");
```

Comment:
Is there an advantage of moving the value into decode (i.e., of removing the `&`)?

Reply:
It was suggested by clippy